conn.go

```diff
@@ -7,19 +7,24 @@ package zmq4
 import (
 	"bytes"
 	"encoding/binary"
+	"errors"
 	"io"
+	"net"
 	"strings"
 	"sync"
+	"sync/atomic"
 
 	"golang.org/x/xerrors"
 )
 
+var ErrClosedConn = errors.New("read/write on closed connection")
```

Comment:
Let's use `xerrors` consistently (it will be easier to remove it once Go-1.13 is the last supported version)

```suggestion
var ErrClosedConn = xerrors.New("zmq4: read/write on closed connection")
```